packages/x-charts/src/ChartContainer/useChartContainerProps.ts

```diff
@@ -19,7 +22,7 @@ export const useChartContainerProps = <
 >(
   props: ChartContainerProps<TSeries, TSignatures>,
   ref: React.Ref<SVGSVGElement>,
-): UseChartContainerPropsReturnValue<TSeries> => {
+): UseChartContainerPropsReturnValue<TSeries, TSignatures> => {
```

Comment:
The hack I found is to force the type as `AllPlugins` but export the DataProvideProps as `TSignatures`